plugins/modem-manager/fu-mm-device.c

```diff
@@ -130,6 +154,17 @@ fu_mm_device_probe_default (FuDevice *device, GError **error)
 				     "modem cannot be put in programming mode");
 		return FALSE;
 	}
+	firmware_update_str = mm_modem_firmware_update_method_build_string_from_mask (self->update_methods);
+
+	if (!validate_firmware_update_method (self->update_methods)) {
+		g_set_error (error,
+			     FWUPD_ERROR,
+			     FWUPD_ERROR_NOT_SUPPORTED,
+			     "invalid firmware update combination: %s",
+			     firmware_update_str);
+		return FALSE;
```

Comment:
could you move this to validate_firmware_update_method() perhaps? Then it returns gboolean and GError like a lot of other things in fwupd. Ta. Then LGTM.

Reply:
That's a nice change, done now

Reply:
Done